scripts/github_pr/interactions/default.rb

```diff
@@ -0,0 +1,156 @@
+#
+# Github_PR default interactions (filter and action classes)
+#
+# github_pr.rb first requires this default.rb file,
+# then all files in the interaction_dirs that match "ia_*.rb" (ia = interaction)
+# The interaction_dirs can be defined in the yaml config as:
+#   interaction_dirs:
+#     - ./dir1
+#     - /other/dir2
+# or as command line parameter
+#   --interaction_dirs ./dir1,/other/dir2
+#
+
+module GithubPR
+  class Interaction
+    def initialize(ghc_ctx, config = {})
+      @ghc_ctx = ghc_ctx
```

Comment:
What is `ghc_ctx`?

Reply:
Its the context that Octokit needs.
Not to be mixed up with the `context` of a commit, eg. `suse/mkcloud`.
Therefor I called it `ghc_ctx` (for **g**it**h**ub **c**lient **c**on**t**e**x**t).
I will add a short comment about it.